### ISSUE TITLE
Spurious error when using --isolatedModules in development

### ISSUE THREAD

**Issue by bwbuchanan** [typed-ember/glint#455]
https://github.com/typed-ember/glint/blob/e35714c5fbccf279eaf476dd6e0b2161397085d2/packages/environment-ember-loose/-private/index.ts

This file needs `export {};` to avoid a spurious error when typechecking a project that uses `--isolatedModules`.

```
../../../repos/glint/packages/environment-ember-loose/-private/index.ts:1:1 -
error TS1208: 'index.ts' cannot be compiled under '--isolatedModules' because it is considered a global
script file. Add an import, export, or an empty 'export {}' statement to make it a module.
```

This issue only affects use of a directly-linked glint repo (e.g. during development / testing of glint itself using a different project as the subject). The generated `index.d.ts` does contain the `export {};`, so users of the npm package are unaffected.

**Comment by bwbuchanan** [comment]
`environment-ember-template-imports` and `environment-glimmerx` are also affected.
